demo/src/index.tsx

```diff
@@ -3,4 +3,11 @@ import ReactDOM from 'react-dom'
 import './index.css'
 import App from './App'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+)
+
+// ReactDOM.render(<App />, document.getElementById('root'))
```

Comment:
Lets get rid of this comment. It's probably better to make sure the library works in Strict Mode.